commercetools-java-client-core/src/main/java/io/sphere/sdk/client/SphereClientFactory.java

```diff
@@ -55,6 +55,22 @@ default SphereClient createClient(String projectKey, String clientId, String cli
         return createClient(SphereClientConfig.of(projectKey, clientId, clientSecret));
     }
 
+
+    /**
+     * Creates a standard client suitable for online shops.
+     *
+     * For the credentials consult <a href="https://admin.sphere.io">the Merchant Center</a>.
```

Comment:
I would suggest to reference merchant center here, because it already supports creation of api clients.

Reply:
nice catch 👍